clang/test/Modules/pr59999.cppm

```diff
@@ -12,44 +12,44 @@
 // RUN:     -fmodule-file=Module=%t/Module.pcm -emit-llvm -o - | FileCheck %t/Object.cppm
 
 // Test again with reduced BMI.
-// RUN: rm -rf %t
-// RUN: mkdir -p %t
-// RUN: split-file %s %t
+// RUNX: rm -rf %t
```

Comment:
are these leftovers from previous versions of the patch that need to be reverted?

Reply:
Oh, sorry. This is something left during debugging. I will remove it.